blueprints/getting-started/README.md

```diff
@@ -33,7 +41,18 @@ For more detailed information, see the documentation for the [Terraform Core wor
 
 On successful completion, Terraform will display outputs containing URLs to the AWS Console as well as `kubectl`-specific commands.
 
-<!-- TODO: Fix -->
+After deploying the mesh, deploy HashiCups demo app.
+
+Demo can be found here: https://github.com/hashicorp-education/learn-consul-get-started-kubernetes/tree/main/self-managed/eks/hashicups
+
+After deploying the mesh, deploy HashiCups demo app.
+
+Demo can be found here: https://github.com/hashicorp-education/learn-consul-get-started-kubernetes/tree/main/self-managed/eks/hashicups
```

Comment:
@bukunmig - you've pasted the fix twice also look at the readme format here to hide long URLs

https://anvilproject.org/guides/content/creating-links

Reply:
also, please make it a single line like line 58 below, thanks!